packages/pipeline/src/Pipeline.tsx

```diff
@@ -0,0 +1,285 @@
+import React, {
+  forwardRef,
+  createRef,
+  useState,
+  Ref,
+  ReactElement,
+  ReactNode,
+} from 'react';
+
+import PropTypes from 'prop-types';
+import { findLast } from 'lodash';
```

Comment:
```suggestion
import findLast from 'lodash/findLast';
```

We really need to install that babel plugin so we don't have to worry about this...

Reply:
Resolved by 022bad9